### ISSUE TITLE
RFI: Trophy

### ISSUE THREAD

**Issue by chrismessina** [feathericons/feather#157]
- icon name: Trophy 🏆
- at least one use case: achievement
- screenshots of similar icons

![np_trophy_1124316_000000](https://user-images.githubusercontent.com/22610/28484772-35dd9338-6e29-11e7-8996-b4b29e9e3891.png)

![np_trophy_925627_000000](https://user-images.githubusercontent.com/22610/28484785-484455a2-6e29-11e7-908b-45dc2c3bb2ee.png)

**Comment by LukyVj** [comment]
Hello! I was also looking for a similar icon. 

Two years later, is the issue still gonna get taken in consideration, or should I try to look for an other icon from feather? 

Thanks tho, we at @algolia love your icon set :)

**Comment by locness3** [comment]
@LukyVj Maybe someone you know can do it and submit it ? Guide can be found in #171

**Comment by MelonCola** [comment]
Idk anyone who can make this, but I feel like it's one of the "essential icons" and really shouldn't be forgotten about. Would be really useful for me.

**Comment by cngeru** [comment]
Still waiting on this

**Comment by chrismessina** [comment]
Hey @colebemis — I went ahead and made some variations for the Trophy icon. LMKWYT.

![variant A](https://user-images.githubusercontent.com/22610/84087437-f21af000-a99e-11ea-8562-caf929fc63f1.png) ![variant B](https://user-images.githubusercontent.com/22610/84087439-f2b38680-a99e-11ea-9691-00245ab72ed0.png) ![variant C](https://user-images.githubusercontent.com/22610/84087440-f2b38680-a99e-11ea-8511-f7f6819bcf31.png) ![variant D](https://user-images.githubusercontent.com/22610/84087441-f34c1d00-a99e-11ea-9925-a145aa9905b2.png) ![variant E](https://user-images.githubusercontent.com/22610/84087443-f34c1d00-a99e-11ea-80d6-35f89aefa978.png)


[svgs.zip](https://github.com/feathericons/feather/files/4748950/svgs.zip)

**Comment by cngeru** [comment]
When will they be added to the icon list

**Comment by chrismessina** [comment]
That's up to @colebemis.

( https://chrismessina.me )

Chris Messina ( https://chrismessina.me ) Founder  ◦  Speaker  ◦  Product Designer Previously: Google, Uber, Y Combinator W'18 🏆 #1 Product Hunter ( https://www.producthunt.com/@chrismessina ) ( https://instagram.com/chris ) ( https://twitter.com/@chrismessina ) ( https://linkedin.com/in/factoryjoe/ ) ( https://medium.com/@chrismessina ) ( https://www.producthunt.com/@chrismessina )

Book a 30 minute call with me ( https://superpeer.com/chrismessina )

**Comment by locness3** [comment]
The request is still valid, it hasn't been added

**Comment by cngeru** [comment]
@colebemis

**Comment by jweled** [comment]
Two years later and it still hasn't been added?

**Comment by locness3** [comment]
No, it has however been added to the community fork @lucide-icons : https://lucide.dev/icon/trophy?search=trophy
